src/elm/State.elm

```diff
@@ -24,13 +24,16 @@ initModel =
     }
 
 
+port begin : () -> Cmd msg
+
+
 init : Navigation.Location -> ( Model, Cmd Msg )
 init location =
     let
         model =
             viewFromUrl location initModel
     in
-        model ! []
+        model ! [ begin () ]
```

Comment:
Can we name this something more descriptive?

Reply:
Done!